### ISSUE TITLE
Windows nightly build failure

### ISSUE THREAD

**Issue by peterjc123** [pytorch/vision#1484]
```
  D:/_work/1/s/packaging/windows/conda/envs/py35/lib/site-packages/torch/include\c10/util/BFloat16.h(48): warning: calling a __host__ function from a __host__ __device__ function is not allowed
  
  D:/_work/1/s/packaging/windows/vision/torchvision/csrc/cuda/PSROIAlign_cuda.cu(339): error: no instance of function template "at::cuda::ATenCeilDiv" matches the argument list
              argument types are: (int64_t, long)
  
  D:/_work/1/s/packaging/windows/vision/torchvision/csrc/cuda/PSROIAlign_cuda.cu(339): error: no instance of overloaded function "std::min" matches the argument list
              argument types are: (<error-type>, long)
  
  D:/_work/1/s/packaging/windows/vision/torchvision/csrc/cuda/PSROIAlign_cuda.cu(398): error: no instance of function template "at::cuda::ATenCeilDiv" matches the argument list
              argument types are: (int64_t, long)
  
  D:/_work/1/s/packaging/windows/vision/torchvision/csrc/cuda/PSROIAlign_cuda.cu(398): error: no instance of overloaded function "std::min" matches the argument list
              argument types are: (<error-type>, long)
  
  4 errors detected in the compilation of "C:/Users/VssAdministrator/AppData/Local/Temp/tmpxft_00000634_00000000-14_PSROIAlign_cuda.compute_70.cpp1.ii".
  PSROIAlign_cuda.cu
  error: command 'C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.2\\bin\\nvcc.exe' failed with exit status 1
```

**Comment by peterjc123** [comment]
related pr: https://github.com/pytorch/vision/pull/1410

**Comment by fmassa** [comment]
Fixed by #1485
